letsencrypt-auto

```diff
@@ -19,13 +19,20 @@ else
   SUDO=
 fi
 
-for arg in "$@" ; do 
+for arg in "$@" ; do
   # This first clause is redundant with the third, but hedging on portability
   if [ "$arg" = "-v" ] || [ "$arg" = "--verbose" ] || echo "$arg" | grep -E -- "-v+$" ; then
     VERBOSE=1
   fi
 done
 
+if [ -f /etc/debian_version ] ; then
+    # squeeze needs Python 2.6, `python2` doesn't exist
+    if grep -q '^6\.0' /etc/debian_version ; then
```

Comment:
I think it's safer to test for the actual prevailing conditions that cause the problem rather than an OS version stamp that *should* indicate it.  It's too common for people to have a mix of different packages on their systems.

So check for something like: no `python2` or `python2.7`, `python` exists but `python --version` indicates 2.6?